package/batocera/emulators/retroarch/libretro/libretro-ecwolf/libretro-ecwolf.mk

```diff
@@ -0,0 +1,18 @@
+################################################################################
+#
+# ECWOLF
+#
+################################################################################
+# Version.: Commits on Jun 19, 2021
+LIBRETRO_ECWOLF_VERSION = a2c0d7bd158ef086adcd78d498df19233d5ba0c7
+LIBRETRO_ECWOLF_SITE = $(call github,libretro,ecwolf,$(LIBRETRO_ECWOLF_VERSION))
+LIBRETRO_ECWOLF_LICENSE = Non-commercial
+
+LIBRETRO_ECWOLF_PLATFORM = $(LIBRETRO_PLATFORM)
+
```

Comment:
LIBRETRO_ECWOLF_BUILD_CMDS is missing

Reply:
Have add LIBRETRO_ECWOLF_BUILD_CMDS thank you.